cliquet/resource/viewset.py

```diff
@@ -28,12 +28,27 @@ class ViewSet(object):
 
     readonly_methods = ('GET', 'OPTIONS', 'HEAD')
 
+    content_types = ["application/json"]
```

Comment:
Since this is not used at the class level, I would suggest not to define it here but in a constant (repeating is even acceptable IMO). 

What do you think?